bip-musig2/reference.py

```diff
@@ -303,7 +306,7 @@ def get_session_values(session_ctx: SessionContext) -> Tuple[Point, int, int, in
 
 def get_session_key_agg_coeff(session_ctx: SessionContext, P: Point) -> int:
     (_, pubkeys, _, _, _) = session_ctx
-    return key_agg_coeff(pubkeys, cbytes(P))
+    return key_agg_coeff(pubkeys, PlainPk(cbytes(P)))
```

Comment:
Just checking, is there any possible reason we'd want to make `cbytes` return a `PlainPk` directly (and make all the other resulting changes needed)?

Reply:
cbytes is also used for pubnonces and not only for `PlainPk`.